code/modules/projectiles/boxes_magazines/external/rifle.dm

```diff
@@ -81,12 +81,37 @@
 	icon_state = "ak47_mag-[hasammo]"
 
 /obj/item/ammo_box/magazine/ebr
-	name = "\improper M514 EBR Magazine (.308 Winchester FMJ)"
+	name = "\improper M514 EBR Magazine (.308)"
 	icon_state = "ebr_mag"
-	ammo_type = /obj/item/ammo_casing/win308
-	caliber = ".308 Winchester FMJ"
+	ammo_type = /obj/item/ammo_casing/a308
+	caliber = ".308"
 	max_ammo = 10
 
+/obj/item/ammo_box/magazine/ebr/update_icon()
+	..()
```

Comment:
you sure you dont wanna call parent?
also both of those reviews for the other mag too